Source/Diagnostics/BTDiagnostics.H

```diff
@@ -399,8 +399,6 @@ private:
         lab-frame data. */
     void InitializeParticleFunctors () override;
 
-    /** Update total number of particles flushed for all species for ith snapshot */
-    void UpdateTotalParticlesFlushed(int i_buffer);
```

Comment:
Removed and delegated to the I/O backend, which currently does the filtering and thus knows what number of particles is truly written.

Reply:
is this also done for plotfile?

Reply:
ah I see - its not needed for plotfile - because we just merge headers and interleave data . this was needed mainly for openpmd. 
all good!